index.js

```diff
@@ -149,6 +149,32 @@ app.post("/user_signup", async (req, res) => {
   }
 });
 
+app.post("/user_auth", async (req, res) => {
+  var reqBody = req.body;
+  var check_auth_user_info_by_email = await get_auth_user_info_by_email(
```

Comment:
rename variable - the semantics are not obvious. Eg: use something like "user_exists" 
also, rename function - same problem